pandas/compat/__init__.py

```diff
@@ -38,6 +38,7 @@
 import struct
 import inspect
 from collections import namedtuple
+import collections
```

Comment:
Using ABCs from `collections` is deprecated. Need to use `collections.abc` instead (which isn't in Py2).

Reply:
Assuming the subsequent changes didn't generate any failures does this point to a potential lack of compatibility test coverage?

Reply:
I don't follow 100%, but I don't think so. On 3.7, you basically couldn't import pandas when `-W error:DeprecationWarning` is set, so this was definitely being hit..